### DISCUSSION TITLE
Timelapse Photo with QRControl App works on hero 8 but not on hero 12

### DISCUSSION THREAD

**Discussion by Firewoods** [gopro/labs#1178] [Q&A]
Dear community, 
I used a QR code for my GoPro hero 8 with GoPro Labs. The code is generated with the QRControl App (updated 2024/09/19). It works quite fine. On my hero 12 the QR code does not work in the same way. I use the latest Labs hero12 02.30.70 UPDATE. Presenting the QR code to the hero 12 the checkbox appears and it then starts recording a video where I want it to start recording Photos all 30 seconds. Does anyone have an idea how to solve it or what I might do wrong? 
Thanks for any help.

**Comment by dnewman-gpsw** [comment]
You do need to share you QR Code or the command string. But guessing I think you are wanting a timelapse Photo (TLP) mode, and the camera doesn't have a TLP preset by default, so you need to create that first. Documentation in QRControl
![image](https://github.com/user-attachments/assets/28703c62-bc0f-43cc-94fa-3dae175eb2b0)

**Reply by Firewoods** [reply]
Thank you so much! That was it! Now it works. With adding a new preset for the the photo time lapse as described here: https://community.gopro.com/s/article/HERO12-Black-Time-Lapse-Settings-And-Resolutions?language=en_US#timelapsephoto
